cmake/ngraph.cmake

```diff
@@ -0,0 +1,53 @@
+include(ExternalProject)
+
+add_library(ngraph_interface_lib INTERFACE)
+
+set(EXTERNAL_NGRAPH_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/3rdparty/ngraph)
+set(EXTERNAL_NGRAPH_SOURCE_DIR ${CMAKE_SOURCE_DIR}/3rdparty/ngraph)
+
+
+ExternalProject_Add(
+	ext_ngraph
+	SOURCE_DIR ${EXTERNAL_NGRAPH_SOURCE_DIR}
+
+	PREFIX 3rdparty/ngraph/
+	CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+			-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+			-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+			-DCMAKE_INSTALL_PREFIX=${EXTERNAL_NGRAPH_INSTALL_PREFIX}
```

Comment:
Might want to set additional cmake options for ngraph.  In particular, `NGRAPH_USE_PREBUILT_LLVM`.